lib/v2/blocs/authentication/viewmodels/authentication_bloc.dart

```diff
@@ -48,5 +49,15 @@ class AuthenticationBloc extends Bloc<AuthenticationEvent, AuthenticationState>
     if (event is DisableBiometric) {
       settingsStorage.biometricActive = false;
     }
+    if (event is OnLogout) {
+      // copy account before clear data
+      var account = settingsStorage.accountName;
+      // Clear data
+      settingsStorage.removeAccount();
+      // User logout --> re-start auth status
+      add(const InitAuthStatus());
+      // Remove fcm token must be last instruction to allow logout, even if there is an error here.
+      await FirebaseMessageTokenRepository().removeFirebaseMessageToken(account);
```

Comment:
is FirebaseMessageTokenRepository part of v2 already?

Reply:
You right i need move this.